homeassistant/components/otbr/config_flow.py

```diff
@@ -13,6 +16,22 @@ class OTBRConfigFlow(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
+    async def async_step_user(
+        self, user_input: dict[str, str] | None = None
+    ) -> FlowResult:
+        """Set up by user."""
+        if self._async_current_entries():
+            return self.async_abort(reason="single_instance_allowed")
```

Comment:
Tests for this are missing. We should keep coverage for integration @ 100%

Reply:
Oops! I'll fix that in <https://github.com/home-assistant/core/pull/86097>

Reply:
Awesome! 👍